stix_shifter_modules/rhacs/configuration/lang_en.json

```diff
@@ -1,24 +1,32 @@
 {
-    "connection": {
-        "host": {
-            "label": "Management IP address or Hostname",
-            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
-        },
-        "port": {
-            "label": "Host Port",
-            "description": "Set the port number that is associated with the Host name or IP"
-        },
-        "help": {
-            "label": "Need additional help?",
-            "description": "More details on the data source setting can be found in the specified link"
-        }
+  "connection": {
+    "host": {
+      "label": "Management IP address or Hostname",
+      "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
     },
-    "configuration": {
-        "auth": {
-            "token": {
-                "label": "API Token",
-                "description": "Token with readonly access to the alert API"
-            }
-        }
+    "port": {
+      "label": "Host Port",
+      "description": "Set the port number that is associated with the Host name or IP"
+    },
+    "help": {
+      "label": "Need additional help?",
+      "description": "More details on the data source setting can be found in the specified link"
+    },
+    "sni": {
+      "label": "Server Name Indicator",
+      "description": "The Server Name Indicator (SNI) enables a separate hostname to be provided for SSL authentication"
+    },
+    "selfSignedCert": {
+      "label": "RHACS Connection Certificate",
+      "description": "Use self-signed SSL certificate and CA content(root and intermediate) of data source"
```

Comment:
Wouldn't this be SSL certificate or CA certificate? The wording makes it sounds like both are suppose to be included here.

Reply:
Updated the text with "or" for better understanding.